code/posix_platform.cpp

```diff
@@ -1,42 +1,3 @@
-// Copyright (c) 2016-2017 Andrew Kallmeyer <fsmv@sapium.net>
```

Comment:
Make sure to keep the copyright notice in this file too

Reply:
Updated in 8e80d67

Reply:
Seems this fix got missed